gradle.properties

```diff
@@ -24,4 +24,4 @@ systemProp.knownSparkVersions=2.4,3.0,3.1,3.2
 systemProp.defaultScalaVersion=2.12
 systemProp.knownScalaVersions=2.12,2.13
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx768m
+org.gradle.jvmargs=-Xmx1g
```

Comment:
You might consider changing to the following, so that the number of classes loaded in memory will be capped (unused classes will be unloaded). This has helped CI in both Spark, as well as some things I've used elsewhere).

```
org.gradle.jvmargs=-Xmx1g -XX:MaxMetaspaceSize=512m
```